tests/utils/verification/volumeproperties.go

```diff
@@ -173,7 +173,7 @@ func VerifyDetachedStatus(name, hostName, esxName string) bool {
 	log.Printf("Confirming detached status for volume [%s]\n", name)
 
 	//TODO: Need to implement generic polling logic for better reuse
-	for attempt := 0; attempt < 30; attempt++ {
+	for attempt := 0; attempt < 60; attempt++ {
```

Comment:
I've already added TODO here. We need a general poller util to avoid blind sleeping in the tests - see issue #1301.